pubsub/listener.go

```diff
@@ -101,8 +102,8 @@ func (listener *EventsListener) ListenForMessages() {
                     request.GetEvent().GetTransition().GetEvent(), err)))
             continue
         }
-        listener.logger.Info("Event `%s` transitioned FSM [%s] to state `%s` - updating store",
-            request.Event.Transition.Event, smId, fsm.State)
+        listener.logger.Info("Event `%s` transitioned FSM [%s] to state `%s` from state `%s` - updating store",
+            request.Event.Transition.Event, smId, fsm.State, previousState)
```

Comment:
Could we add a log line before we try to "SendEvent" ? It seems like it can fail there and we wouldn't have the IDs logged (SM ID, event ID, previous state)

Reply:
Sure, adding as a debug entry

```
        listener.logger.Debug("Preparing to send event `%s` for FSM [%s] (current state: %s)",
            request.Event.Transition.Event, smId, previousState)
```

Reply:
not only we capture the error if `SendEvent` fails, but we also send a notification (L98)